bin/chown/chown.c

```diff
@@ -0,0 +1,289 @@
+/*	$NetBSD: chown.c,v 1.8 2012/10/24 01:12:51 enami Exp $	*/
```

Comment:
Have you changed anything in this file or is it taken verbatim from _NetBSD_?

Reply:
Almost nothing. This is whole diff:
```
33,45d32
< #ifndef lint
< __COPYRIGHT("@(#) Copyright (c) 1988, 1993, 1994, 2003\
<  The Regents of the University of California.  All rights reserved.");
< #endif /* not lint */
<
< #ifndef lint
< #if 0
< static char sccsid[] = "@(#)chown.c   8.8 (Berkeley) 4/4/94";
< #else
< __RCSID("$NetBSD: chown.c,v 1.8 2012/10/24 01:12:51 enami Exp $");
< #endif
< #endif /* not lint */
<
66c53
< __dead static void    usage(void);
---
> static void   usage(void);
```